### ISSUE TITLE
[BUG] ValueError: invalid literal for int() with base 10: b'' - Anomaly Detection Jupyter Example

### ISSUE THREAD

**Issue by na018** [salesforce/Merlion#92]
**Describe the bug**
Running the example Jupyter Notebook Merlion/examples/anomaly/0_AnomalyIntro.ipynb (second sell) - Model can not be trained
Java was installed as described: 
```
➜  ~ java -version
java version "1.8.0_311"
Java(TM) SE Runtime Environment (build 1.8.0_311-b11)
Java HotSpot(TM) 64-Bit Server VM (build 25.311-b11, mixed mode)
```
**To Reproduce**
Running the example Jupyter Notebook Merlion/examples/anomaly/0_AnomalyIntro.ipynb (second sell)

**Expected behavior**
Model can be trained as expected.

**Screenshots**
<img width="1184" alt="image" src="https://user-images.githubusercontent.com/23548766/170502249-24801239-b20b-445c-8492-a0b517d47809.png">
<img width="1244" alt="image" src="https://user-images.githubusercontent.com/23548766/170502317-136abd90-7ac2-458e-881d-fd3e140b13ba.png">

**Desktop (please complete the following information):**
 - OS: [MacOS Monterey 12.3.1]
 - Merlion Version [ 1.1.3]

**Additional context**

**Comment by na018** [comment]
The error was introduced in https://github.com/salesforce/Merlion/pull/62 from @aadyotb  
This version still works a0afb87 for the example notebooks

**Comment by aadyotb** [comment]
Hi @na018 I'm not able to reproduce the error. Can you check to see if the `$JAVA_HOME` environment variable is set? On Mac OS Monterrey, my `~/.bash_profile` file has the line `export JAVA_HOME=$(/usr/libexec/java_home)`.

**Comment by waue0920** [comment]
Hi, I also encountered the same issue, that is because the old java version 1.8.
It is resolved after I remove openjdk 1.8 and install 1.11. 

$ sudo apt remove openjdk-1.8
$ sudo apt install openjdk-1.11
